nucliadb_models/src/nucliadb_models/search.py

```diff
@@ -506,10 +505,18 @@ class SearchParamDefaults:
     )
     top_k = ParamDefault(
         default=20,
+        gt=-1,
```

Comment:
NIT
```suggestion
        ge=0,
```
may be more clear